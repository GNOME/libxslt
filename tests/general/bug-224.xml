<doc>
    <item/>
    <item/>
    <item/>
    <item/>
    <item/>
    <item/>
</doc>
