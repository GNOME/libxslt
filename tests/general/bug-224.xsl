<xsl:stylesheet version="1.0" xmlns:xsl="http://www.w3.org/1999/XSL/Transform">

<xsl:variable name="v1" select="generate-id(doc/item[1])"/>
<xsl:variable name="v2" select="generate-id(doc/item[2])"/>
<xsl:variable name="v3" select="generate-id(doc/item[3])"/>
<xsl:variable name="v4" select="generate-id(doc/item[4])"/>
<xsl:variable name="v5" select="generate-id(doc/item[5])"/>
<xsl:variable name="v6" select="generate-id(doc/item[6])"/>

<xsl:template match="/">
    <result>
        <xsl:text>&#xa;</xsl:text>
        <xsl:value-of select="$v1"/>
        <xsl:text>&#xa;</xsl:text>
        <xsl:value-of select="$v2"/>
        <xsl:text>&#xa;</xsl:text>
        <xsl:value-of select="$v3"/>
        <xsl:text>&#xa;</xsl:text>
        <xsl:value-of select="$v4"/>
        <xsl:text>&#xa;</xsl:text>
        <xsl:value-of select="$v5"/>
        <xsl:text>&#xa;</xsl:text>
        <xsl:value-of select="$v6"/>
        <xsl:text>&#xa;</xsl:text>
    </result>
</xsl:template>

</xsl:stylesheet>
