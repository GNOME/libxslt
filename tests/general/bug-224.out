<?xml version="1.0"?>
<result>
id1
id2
id3
id4
id5
id6
</result>
